src/components/dialogs/affiliation-error.vue

```diff
@@ -23,17 +40,32 @@
 import { Component, Vue } from 'vue-property-decorator'
 import { Action, Getter } from 'vuex-class'
 import { ActionBindingIF } from '@/interfaces/store-interfaces'
+import ContactInfo from '@/components/common/contact-info.vue'
 
-@Component({})
+@Component({
+  components: { ContactInfo }
+})
 export default class AffiliationErrorDialog extends Vue {
-  @Getter getAffiliationErrorModalVisible!: boolean
-  @Action setAffiliationErrorModalVisible!: ActionBindingIF
+  @Getter getAffiliationErrorModalValue!: number
+  @Action setAffiliationErrorModalValue!: ActionBindingIF
+
+  get isUnableToStart (): boolean {
+    return (this.getAffiliationErrorModalValue === 1)
```

Comment:
I didn't think it was worth an enum for this. Thoughts?

0 = hide modal
1 = unable to start registration
2 = associated with another account

Maybe create an enum if we add another value in the future?